dev/com.ibm.ws.webcontainer.cors/test/com/ibm/ws/webcontainer/cors/CorsHelperTest.java

```diff
@@ -56,7 +56,7 @@
     private static final String RESPONSE_HEADER_ACCESS_CONTROL_EXPOSE_HEADERS = "Access-Control-Expose-Headers";
 
     /** Optional. Response to preFlight(OPTIONS). Indicates how long the results of a preFlight request can be cached. */
-    private static final String RESPONSE_HEADER_ACCESS_CONTROL_MAX_AGE = "Access-Control-Allow-Max-Age";
+    private static final String RESPONSE_HEADER_ACCESS_CONTROL_MAX_AGE = "Access-Control-Max-Age";
```

Comment:
Please update [copyright header](https://github.com/OpenLiberty/open-liberty/pull/9952/files#diff-6a558ddb6ca3e5a821fcd0281ee1d5fbR2) to `Copyright (c) 2011, 2019 IBM...`